scratch/apple-catcher.html

```diff
@@ -8,47 +8,51 @@
     <div class="container">
         <div class="block">
 
-            <p>In this project, we're going to build a game, where you need to catch falling apples before they hit the ground! </p>
+            <p>In this project, we're going to build a game where you need to catch falling apples before they hit the ground! </p>
             <iframe allowtransparency="true" width="485" height="402" src="//scratch.mit.edu/projects/embed/94880040/?autostart=false" frameborder="0" allowfullscreen></iframe>
         </div>
         <div class="block">
-            <p>1) Choose a background and two sprites: one for your main character and another one to catch.</p>
-            <iframe allowtransparency="true" width="485" height="402" src="//scratch.mit.edu/projects/embed/90970675/?autostart=false" frameborder="0" allowfullscreen></iframe>
+            <p>1) Choose a background and a sprite for your main character.</p>
+            <iframe allowtransparency="true" width="485" height="402" src="//scratch.mit.edu/projects/embed/152162379/?autostart=false" frameborder="0" allowfullscreen></iframe>
         </div>
         <div class="block">
-            <p>2) Have your character move left and right along the bottom of the screen when the left and right arrow keys are pressed.</p>
-            <iframe allowtransparency="true" width="485" height="402" src="//scratch.mit.edu/projects/embed/109188265/?autostart=false" frameborder="0" allowfullscreen></iframe>
+            <p>2) Make your character move left when you press the left arrow key.</p>
+            <iframe allowtransparency="true" width="485" height="402" src="//scratch.mit.edu/projects/embed/152162674/?autostart=false" frameborder="0" allowfullscreen></iframe>
         </div>
         <div class="block">
-            <p>3) Have the apple start at a random spot on the top of the screen. </p>
+            <p>3) Make your character move right when you press the right arrow key.</p>
+            <iframe allowtransparency="true" width="485" height="402" src="//scratch.mit.edu/projects/embed/152163494/?autostart=false" frameborder="0" allowfullscreen></iframe>
+        </div>
+        <div class="block">
+            <p>4) Create a sprite that your character will have to catch. Make it start at a random spot on the top of the screen when the flag is clicked. </p>
             <iframe allowtransparency="true" width="485" height="402" src="//scratch.mit.edu/projects/embed/126528142/?autostart=false" frameborder="0" allowfullscreen></iframe>
         </div>
         <div class="block">
-            <p>4) Make the apple fall. </p>
+            <p>5) Make the apple fall all the time. </p>
```

Comment:
OK. Tell me what you think now -- with steps four and five combined, we have information about sequence of events ("...when the flag is clicked" and "once the apple appears at a random spot...")

When I've seen students have trouble with this one, they tend to get stuck with translating the word "fall" to the concept "move," and understanding that it should always fall (using a forever)